core/server/translations/en.json

```diff
@@ -251,6 +251,10 @@
                 "memberAlreadyExists": {
                     "message": "Member already exists",
                     "context": "Attempting to add member with existing email address."
+                },
+                "memberAlreadyExistsWithEmail": {
+                    "message": "Different member already exists with edited email",
+                    "context": "Attempting to edit member email to an existing email address."
```

Comment:
@imox2 would rather reuse the error message above - `memberAlreadyExists`. To make it work for both add/edit actions can make the `add` word in the context a parameter. This way wording would stay more consistent and less copy to maintain in the future. For reference, you could check how parameterized errors are used [here](https://github.com/TryGhost/Ghost/blob/c8a6939b2edf8a4e4bdd705f4158be919713c57f/core/server/translations/en.json#L445-L450).

Reply:
@naz thanks for the review. Okay, doing that